feature-libs/product-configurator/rulebased/components/attribute/types/drop-down/configurator-attribute-drop-down.component.ts

```diff
@@ -26,4 +27,8 @@ export class ConfiguratorAttributeDropDownComponent
   ngOnInit() {
     this.attributeDropDownForm.setValue(this.attribute?.selectedSingleValue);
   }
+
+  getSelectedValue(): Configurator.Value | undefined {
+    return this.attribute?.values?.find((value) => value?.selected);
```

Comment:
```suggestion
    return this.attribute.values?.find((value) => value?.selected);
```
attribute is mandatory

Reply:
Done!